packages/algorand-js/src/lib/helpers.ts

```diff
@@ -1,4 +1,34 @@
+import { TealError } from "../errors/errors";
+import { ERRORS } from "../errors/errors-list";
+
+/**
+ * Description: Compare Uint8Arrays
+ * @param a Uint8Array
+ * @param b Uint8Array
+ */
 export function compareArray (a: Uint8Array, b: Uint8Array): Boolean {
   return a.length === b.length &&
     a.every((v, i) => v === b[i]);
 }
+
+/**
+ * Description: assert if string contains digits only
+ * "123" // ok.  "12+2" // error.
+ * @param val : string
+ */
+export function assertOnlyDigits (val: string): void {
+  if (!(/^\d+$/.test(val))) {
```

Comment:
let's move the regexp to a const value in the module scope instead of recreating it every time in the function call:
```
const reDigit = /^\d+$/;
```

Reply:
ok